### PR TITLE
🔊 Add `VERBOSE_AXIOS_LOGGING` Flag to `consent-ui`

### PR DESCRIPTION
**Pull request by mistryrn**
### Consent UI
* Adds a new flag `VERBOSE_AXIOS_LOGGING` which enables logging of Axios requests and responses in `consent-ui`
* Removes `clientSideRootUrl` checks from proxy route handlers

### Special Instructions
In order to enable verbose Axios logging in `consent-ui`, set the following environment variable in `consent-ui/.env.local`:
```.env
VERBOSE_AXIOS_LOGGING=true
```

## PR Readiness Checklist

- [ ] "Expected Outcome(s)" in ticket have been met
- [ ] Ticket number included in PR title
- [ ] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

### CODE REVIEW

File: README.md
Comment (mistryrn) [inline README.md:1]: Mostly whitespace changes/formatting for the env var table, actual change highlighted in next comment.

File: README.md [hunk 1]
```diff
@@ -70,40 +70,41 @@ To run the setup locally, ensure you have provided the **required** environment
...
+| `consent-ui`  | `TOKEN_MAX_AGE`         | Next-Auth session lifespan, in seconds.                                                                                                                                                                                                                 | `number`  | Optional | `3600`                                                                                                                                                         |
+| `consent-ui`  | `VERBOSE_AXIOS_LOGGING` | Enables verbose logging of Axios requests in the consent-ui application.                                                                                                                                                                                | `boolean` | Optional | `false`                                                                                                                                                        |
```
Comment (mistryrn) [inline README.md:96]: This `VERBOSE_AXIOS_LOGGING` env var is the only real change to the table here.

File: apps/consent-ui/src/app/api/[...proxy]/route.ts [hunk 2]
```diff
@@ -36,12 +35,7 @@ const handler = async (
 	req: NextRequest,
 	routePaths: { params: { proxy: string[] } }, // "proxy" key matches the [...proxy] dynamic path
 ): Promise<NextResponse<any>> => {
-	const { CONSENT_API_URL, CONSENT_UI_URL } = getAppConfig();
-	const clientSideRootUrl = urlJoin(CONSENT_UI_URL, PROXY_API_PATH);
```
Comment (mistryrn) [inline apps/consent-ui/src/app/api/[...proxy]/route.ts:40]: Removed as unnecessary, caused errors when running in containerized environments due to mismatch between `CONSENT_UI_URL` and container `HOSTNAME`.

File: apps/consent-ui/src/services/api/axiosClient.ts [hunk 3]
```diff
@@ -19,14 +19,32 @@
...
+	axiosErrorInterceptor,
+	axiosRequestInterceptor,
+	axiosResponseInterceptor,
+} from 'src/services/api/utils';
 
-const initAxiosClient = () => {
-	const { CONSENT_UI_URL } = getAppConfig();
-	return axios.create({
+const { CONSENT_UI_URL, VERBOSE_AXIOS_LOGGING } = getAppConfig();
+const AXIOS_CLIENT_NAME = 'axiosClient';
```
Comment (anncatton) [inline apps/consent-ui/src/services/api/axiosClient.ts:29]: 👍

File: apps/consent-ui/src/services/api/axiosProxyClient.ts [hunk 4]
```diff
@@ -19,14 +19,32 @@
...
+	axiosErrorInterceptor,
+	axiosRequestInterceptor,
+	axiosResponseInterceptor,
+} from 'src/services/api/utils';
 
-const initAxiosClient = () => {
-	const { CONSENT_API_URL } = getAppConfig();
-	return axios.create({
+const { CONSENT_API_URL, VERBOSE_AXIOS_LOGGING } = getAppConfig();
+const AXIOS_CLIENT_NAME = 'axiosProxyClient';
```
Comment (anncatton) [inline apps/consent-ui/src/services/api/axiosProxyClient.ts:29]: 👍

**Review by anncatton — APPROVED** [review]
Grazie! 🚀

**Pull request merged**